src/opensles/OutputMixerOpenSLES.h

```diff
@@ -50,7 +50,8 @@ class OutputMixerOpenSL {
     std::mutex            mLock;
     int32_t               mOpenCount = 0;
 
-    SLObjectItf           mOutputMixObject = 0;
+    SLObjectItf           mOutputMixObject = 0;  // this is not consistent with
+    // the code in the .cpp file, please use nullptr everywhere.
```

Comment:
Agree. This was copied from some very old code.
Needs cleanup.

Reply:
Fixed in #578